tests/test_cloudpath_manipulation.py

```diff
@@ -33,20 +35,26 @@ def test_no_op_actions(rig):
     assert path.is_absolute()
 
 
-def test_relative_to(rig):
-    assert rig.create_cloud_path("a/b/c/d.file").relative_to(
-        rig.create_cloud_path("a/b/")
-    ) == PurePosixPath("c/d.file")
+def test_relative_to(rig, azure_rig, gs_rig):
+    assert rig.create_cloud_path("bucket/path/to/file.txt").relative_to(
+        rig.create_cloud_path("bucket/path")
+    ) == PurePosixPath("to/file.txt")
     with pytest.raises(ValueError):
         assert rig.create_cloud_path("bucket/b/c/d.file").relative_to(
             rig.create_cloud_path("bucket/z")
         )
     with pytest.raises(ValueError):
         assert rig.create_cloud_path("a/b/c/d.file").relative_to(PurePosixPath("/a/b/c"))
+    other_rig = azure_rig if rig.cloud_prefix != azure_rig.cloud_prefix else gs_rig
+    path = rig.create_cloud_path("bucket/path/to/file.txt")
+    other_cloud_path = other_rig.create_cloud_path("bucket/path")
```

Comment:
That's a fair point. I made change. Wouldn't this be a problem for the live_server test though? Since this refers to another bucket

Reply:
I believe that everything should be configured so that instantiation works, and since we a path does not need to exist for the relative_to tests it should be ok.